fe/fe-core/src/main/java/com/starrocks/qe/AuditLogBuilder.java

```diff
@@ -86,10 +86,6 @@ public void exec(AuditEvent event) {
                     continue;
                 }
 
-                if (af.value().equals("Timestamp")) {
```

Comment:
1. Could you name it `StartTime`?
2. Is it a datetime or a timestamp value?

Reply:
the field is defined [here](https://github.com/StarRocks/starrocks/blob/609e40deddc9ed0f4eaf263b20612b0067a102b8/fe/fe-core/src/main/java/com/starrocks/plugin/AuditEvent.java#L72-L73), it's a long unix-timestamp in milliseconds, see the [usages](https://github.com/search?q=repo%3AStarRocks%2Fstarrocks+.setTimestamp%28&type=code).